docs/framework/wcf/migrating-from-net-remoting-to-wcf.md

```diff
@@ -181,7 +181,7 @@ Console.WriteLine(String.Format("  Customer {0} {1} received.",
   
 -   [Client Channel-Level Programming](../../../docs/framework/wcf/extending/client-channel-level-programming.md)  
   
--   [How to: Add, Update, or Remove a Service Reference](../Topic/How%20to:%20Add,%20Update,%20or%20Remove%20a%20Service%20Reference.md)  
+-   [How to: Add, Update, or Remove a Service Reference](http://msdn.microsoft.com/library/cacc14bd-4455-4a44-be78-d2ac16113dd9)  
```

Comment:
Topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/data-tools/how-to-add-update-or-remove-a-wcf-data-service-reference